load-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Locust based load tests for RH-Che/CReW
+
+### Prepare environment
+
+It is recommended to do python-pip installations in a separate python virtual-env
+
+* Installing dependencies:
+    ```
+    yum install python-pip --assumeyes \
+    pip install locust \
+    pip install websocket-client
+    ```
+
+### Running the tests
+
+```
+cd rh-che/load-tests && \
+LOCUST_RHCHE_ACTIVE_TOKEN="<active_token>" \
```

Comment:
Does these variable needs to be "at one line" with the locust start? In other words, is that ```\``` needed at the end of a line?

Reply:
That's for user readability.
The end result is `cd rh-che/load-tests && LOCUST_RHCHE_ACTIVE_TOKEN=<> LOCUST_RHCHE_PROTOCOL=<> LOCUST_RHCHE_BASE_URI=<> locust -f simple.py` which is really long and confusing.
This way the user can easily see the value settings one per line.

Reply:
Ah, I see. I though it is less readable, but if it is more useful, than leave it as it is.

Reply:
But is a difference between what you wrote this approach? 
```
cd rh-che/load-tests
LOCUST_RHCHE_ACTIVE_TOKEN=<> 
LOCUST_RHCHE_PROTOCOL=<> 
LOCUST_RHCHE_BASE_URI=<> 
locust -f simple.py
```

Reply:
Yes. I had issues with `locust -f ...` not seeing all the locally defined variables that way without doing an `export` on them.

Reply:
Ah, ok. So leave it as it is.